client/src/components/atoms/NotificationListItem.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <q-item clickable :class="{ unread: !note.viewed }" class="q-pl-none">
+    <q-badge v-if="!note.viewed" />
+    <q-item-section side class="q-px-md">
+      <q-icon :size="iconSize" :name="note.icon" />
+    </q-item-section>
+    <q-item-section>
+      <p class="q-pa-none q-ma-none">
+        {{ $t(tKey, flattened) }}
+      </p>
+    </q-item-section>
+    <q-item-section
+      v-if="!!showTime"
+      class="text-caption items-end col-md-3 col-xs-1"
+    >
+      {{ relativeTime }}
+    </q-item-section>
+  </q-item>
+</template>
+
+<script>
+import { computed } from "@vue/composition-api"
+import { Screen } from "quasar"
+import { flatten } from "flat"
+import TimeAgo from "javascript-time-ago"
+import en from "javascript-time-ago/locale/en.json"
+
+TimeAgo.addDefaultLocale(en)
+
+const timeAgo = new TimeAgo("en-US")
```

Comment:
Its not a timezone, its a language locale.  This will likely need to get hooked into the translation API at some point for locale switching, but I'd put that low on the list.